### PR TITLE
Feature/network policy

### PR DESCRIPTION
**Pull request by jaxels10**
This is a PR to enable a new feature in Yggdrasil. We need to be able to create networkPolicies. 
Right now, our services folder structure looks like this: yggdrasil/services/namespace/service-name-values-file.yaml. 
I am moving this to make it look like the following: yggdrasil/services/namespace/service-name/service-name-values-file.yaml. 
In the new folder "service-name/", we can add extra files that we need for each namespace. One of these files will be a definition of network policies. This PR enables that. 

**EDIT**
I have edited the _networkPolicy.yaml to include networkPolicy in both the namespace folder and the individual service name folders. This is done because we need to be able to make network policies for both namespaces and services.

### CODE REVIEW

File: chart/templates/_application.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
     helm:
       version: v3
       values: |
-        {{- $valuesFilePath := printf "%s/%s" ($path | dir) .source.valuesFile }}
+        {{- $valuesFilePath := printf "%s/%s/%s" ($path | dir) .name .source.valuesFile }}
```
Comment (klausenbusk) [inline chart/templates/_application.yaml:34]: Where is this variable used?

Comment (jaxels10) [reply chart/templates/_application.yaml:34]: It is used on the next line:  `{{- $valuesFile := $root.Files.Get $valuesFilePath }}`

Comment (klausenbusk) [reply chart/templates/_application.yaml:34]: 🤦

**Review by MadsHT — APPROVED** [review]

**Review by jacobcrawford — APPROVED** [review]
Looks good

**Pull request merged**